artman/cli/main2.py

```diff
@@ -136,13 +136,15 @@ def parse_args(*args):
         help='[Optional] Directory to store output generated by artman. '
         'Default to `./artman-genfiles`', )
     parser.add_argument(
-        '--input-dir',
+        '--root-dir',
         type=str,
-        default='',
+        default=None,
```

Comment:
I got an error previously using a `None` default alongside `type=str`, which is why I made this an empty string. Both are falsy.

Reply:
My bad. Should have changed this.